migration-guides.md

```diff
@@ -0,0 +1,131 @@
+# Migration guides
+
+> :star: Switching from other HTTP request libraries to Got :star:
+
+### Migrating from Request
+
+You may think, it's too hard to switch. But that's not true. Let's take the very first example from `request` and see how it looks using `got`.
+
+```js
+const request = require('request');
+request('http://www.google.com', function (error, response, body) {
+  console.log('error:', error); // Print the error if one occurred
+  console.log('statusCode:', response && response.statusCode); // Print the response status code if a response was received
+  console.log('body:', body); // Print the HTML for the Google homepage.
+});
+
+const got = require('got');
```

Comment:
No point in having both an example using promises and one using async/await. Just drop this on. We should always use async/await.

Reply:
Ok :fire: